bat/BLDCICS.BAT

```diff
@@ -0,0 +1,26 @@
+@if /I "%1" == "tron" (echo on) else (echo off)
+rem rebuild z390 cics support
+
+setlocal
+if /I "%1" == "tron" (set z_TraceMode=tron
+                      shift /1
+              ) else (if /I "%1" == "troff" (set z_TraceMode=troff
+                                             shift /1
+                                     ) else (set z_TraceMode=)
+                      )
+set /A z_NestLevel=%z_NestLevel%+1
+rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
+
+pushd %~dps0..
+call cics\bat\dfhall.bat             %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+call cics\bat\dfhallv.bat            %1 %2 %3 %4 %5 %6 %7 %8 %9 || goto error
+set z_ReturnCode=0
+goto return
+
+:error
+set z_ReturnCode=%ERRORLEVEL%
+echo %0 ERROR: Encountered RC %z_ReturnCode% - exiting
+:return
+popd
+rem ----- Lvl(%z_NestLevel%)  End %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
+exit /b %z_ReturnCode%
```

Comment:
I already recommended that bat/BLDCICS.BAT not be added. Since it is still here, I guess you disagree. Yes? No?